send_rx_functions.php

```diff
@@ -112,8 +145,39 @@ function send_rx_generate_pdf_file($contents, $file_path) {
  * @return bool
  *   TRUE if success, FALSE otherwise.
  */
-function send_rx_update_record_field($project_id, $event_id, $record_id, $field_name, $value) {
-    // TODO.
+function send_rx_update_record_field($project_id, $event_id, $record_id, $field_name, $value, $instance = null) {
```

Comment:
Let's call this function send_rx_save_record_field() and merge it with send_rx_inser_record_field().